storage/innobase/srv/srv0start.cc

```diff
@@ -1403,6 +1404,8 @@ dberr_t srv_start(bool create_new_db)
 		return(srv_init_abort(err));
 	}
 
+	scn_mgr = new SCN_Mgr();
```

Comment:
Can we avoid the pointer indirection and declare `scn_mgr` as a singleton object, like we do for `dict_sys`, `lock_sys`, `buf_pool` and many others?

Reply:
Sure.